doc/src/manual/constructors.md

```diff
@@ -425,7 +425,7 @@ julia> struct OurRational{T<:Integer} <: Real
            end
        end
 
-julia> OurRational(n::T, d::T) where T<:Integer = OurRational{T}(n,d)
+julia> OurRational(n::T, d::T) where {T<:Integer} = OurRational{T}(n,d)
```

Comment:
oh yeah this example's also kinda nonsense, defining a separate `Main.//` with 1 method instead of actually showing how to extend `Base.//`